packages/schematics/angular/ng-new/index_spec.ts

```diff
@@ -103,4 +103,22 @@ describe('Ng New Schematic', () => {
     const { cli } = JSON.parse(tree.readContent('/bar/angular.json'));
     expect(cli.packageManager).toBe('npm');
   });
+
+  it('should experimentalZoneless is set to true and add provideExperimentalZonelessChangeDetection in app.config', async () => {
+    const tree = await schematicRunner.runSchematic('ng-new', {
+      ...defaultOptions,
+      experimentalZoneless: true,
+    });
+    const fileContent = tree.readContent('/bar/src/app/app.config.ts');
+    expect(fileContent).toContain('provideExperimentalZonelessChangeDetection()');
+  });
+
+  it('should experimentalZoneless is set to false', async () => {
```

Comment:
`should not add 'provideExperimentalZonelessChangeDetection()' when 'experimentalZoneless' is false`